src/test/xprocspec/dtbook-to-html.step.xprocspec

```diff
@@ -28,7 +28,7 @@
         <x:expect type="xpath" label="The output document (the intermediary HTML) should have an xml:base attribute with filename C00000.xhtml" test="tokenize(/*/@xml:base,'/')[last()]"
             equals="'C00000.xhtml'"/>
         <x:expect type="xpath" label="The output document (the intermediary HTML) should have an xml:base attribute pointing to the temp-dir directory" test="/*/@xml:base"
-            equals="concat($temp-dir,'temp-dir/C00000.xhtml')"/>
+            equals="concat($temp-dir,'temp-dir/C00000.xhtml')" pending="the base URI is normalized"/>
```

Comment:
Ah, I see now that this is also in this PR, not only in #387. I have the same question here though: why is it pending? Does the xml:base not contain the full path anymore?

Reply:
I don't remember exactly what the URI was that I was getting. It was equivalent (normalized) but could not be compared the way it was done. However I tested it again now and it seems the pending is not needed. Not sure what happened. Can you try to remove it and run the test again?

Reply:
It works for me as well. :+1: